packages/workflow/test/Expression.test.ts

```diff
@@ -79,9 +78,13 @@ describe('Expression', () => {
 			expect(evaluate('={{DateTime.now().toLocaleString()}}')).toEqual(
 				DateTime.now().toLocaleString(),
 			);
+
+			jest.useFakeTimers({ now: new Date() });
 			expect(evaluate('={{Interval.after(new Date(), 100)}}')).toEqual(
```

Comment:
This test was previously passing because both the values being compared were `InvalidInterval`. With the changes in the PR, this code works as expected, but the tests fail now because of the millisecond differences between the used dates. so had to use fake timers here.